app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,6 @@
+class MoviesController < ApplicationController
+  def show
+    @movie = Movie.find(params[:id])
+    @actors = @movie.sort_actors_by_age
```

Comment:
This is fine, and if we want to prioritize thin controllers, you can just call `@movie.sort_actors_by_age` directly in the view instead of setting it equal it an ivar in the controller. That wouldn't break any MVC rules.